keylime/ima/ima_dm.py

```diff
@@ -7,9 +7,10 @@
 import pickle
 import re
 import sys
+import typing
 from collections import ChainMap
 from itertools import chain
-from typing import Dict, List, Optional, Union
+from typing import Any, Callable, Dict, List, Optional, TypeAlias, TypedDict, Union
```

Comment:
```
from typing import Any, Callable, Dict, List, Optional, TypeAlias, TypedDict, Union
ImportError: cannot import name 'TypeAlias' from 'typing' (/usr/lib64/python3.9/typing.py)
```

`New in version 3.10.` according to doc

Reply:
Yeah I missed that this feature is only supported in Python 3.10. Removed it for now.